src/Components/Login.js

```diff
@@ -1,30 +1,139 @@
 import Button from 'react-bootstrap/Button';
 import Form from 'react-bootstrap/Form';
 import 'bootstrap/dist/css/bootstrap.min.css'
+import { useEffect, useState } from 'react';
+import AdminPanel from './adminPanel';
+import { json, useNavigate  } from 'react-router-dom';
```

Comment:
You can import Route and Routes here. Please remove the 7th line.
```suggestion
import { json, useNavigate, Route, Routes  } from 'react-router-dom';
```